documentation/Camara-OAS-Sections-Guidelines.md

```diff
@@ -0,0 +1,108 @@
+# CAMARA OAS Sections Guidelines 
+
+## Table of Contents
+
+   * [Introduction](#introduction)
+   * [Conventions](#conventions)
+   * [OAS Sections](#oas-sections)
+     + [OpenAPI Version](#version)
+     + [Info Object](#info-object)
+       + [Title](#title)
+       + [Description](#description)
+       + [Version](#version)
+       + [Terms of service](#tos)
+       + [Contact information](#contact)
+       + [License](#license)
+       + [Extension field](#ext-field)
+     + [Servers](#servers)
+     + [Paths](#paths)
+     + [Components](#components)
+       + [Schemas ](#schemes)
+       + [Responses](#responses)
+       + [Parameters](#parameters)
+       + [Request bodies](#req-bodies)
+       + [Headers](#headers)
+       + [Security schemes](#sec-schemes)
+     + [Security](#security)
+     + [Tags](#tags)
+     + [External Documentation](#ext-doc)
+
+## Introduction
+Camara uses OpenAPI Specification (OAS) to describe its APIs. The below guidelines specify the restrictions or conventions to be followed within the OAS yaml by all Camara APIs (referred below simply as APIs).
+
+## Conventions
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
```

Comment:
Suggest to add a section on the OAS file name which should be the api name in kebab-case.yaml e.g. simple-edge-discovery.yaml